feeder/src/vote.ts

```diff
@@ -296,19 +296,20 @@ interface VoteArgs {
 
 export async function vote(args: VoteArgs): Promise<void> {
   const client = new LCDClient({
-    URL: args.lcdAddress,
-    chainID: args.chainID,
-    gasPrices: args.gasPrices,
+    URL: args.lcdAddress || process.env['LCD_ADDRESS'],
+    chainID: args.chainID || process.env['CHAIN_ID'],
+    gasPrices: args.gasPrices || process.env['GAS_PRICE'],
   })
-  const rawKey: RawKey = await initKey(args.keyPath, args.password)
-  const valAddrs = args.validator || [rawKey.valAddress]
+  const rawKey: RawKey = await initKey(process.env['KEY_PATH'] || args.keyPath, process.env['PASSPHRASE'] || args.password)
+  const valAddrs = (process.env['VALIDATOR'] && process.env['VALIDATOR'].split(',')) || args.validator || [rawKey.valAddress]
```

Comment:
I think `args` should have higher priority than `env`. See my comments in #31. Priority of `args` vs `env` is applicable to all changes across the codebase. So whatever is done in this pull request needs to have order of `args` first, `env` second.

Reply:
Have made the swap